q2_types/per_sample_sequences/tests/test_transformer.py

```diff
@@ -138,11 +138,11 @@ def test_casava_one_eight_single_lane_per_sample_dirfmt_to_slpssefdf(self):
         for act, exp in zip(obs, input):
             self.assertEqual(act, exp)
 
-    def test_casava_one_eight_single_lane_per_sample_dirfmt_to_slpspefdf(self):
+    def test_slpssefdf_to_casava_one_eight_single_lane_per_sample_dirfmt(self):
         filenames = ('Human-Kneecap_S1_L001_R1_001.fastq.gz',)
         input, obs = self.transform_format(
-            CasavaOneEightSingleLanePerSampleDirFmt,
-            SingleLanePerSamplePairedEndFastqDirFmt, filenames=filenames
+            SingleLanePerSampleSingleEndFastqDirFmt,
+            CasavaOneEightSingleLanePerSampleDirFmt, filenames=filenames
```

Comment:
I think we need a _new_ test. Also, I'm a little surprised that validate isn't failing here, as you should have leftover files post-transform which should be caught by the directory format.

Does `transform_format` do something unusual with respect to the transformation API?